AlgoriParams:
    is_enemy_red: True
    h_min_r: 14
    h_max_r: 240
    h_min_b: 220
    h_max_b: 255
    s_min: 40
    s_max: 255
    v_min: 200
    v_max: 255

CameraParams:
    cx: 630.5
    cy: 495.1
    fx: 871
    fy: 871
